### PR TITLE
Improvements on recent circleci refactor

### PR DESCRIPTION
**Pull request by ajsantander**
#### Changes
* Removes redundant commands. Only two commands remain, and that's because they are either used by more than a single job, or because encapsulating functionality in them makes jobs more readable.
* Adds a new workflow-scheduled that runs on a timer. Note that this workflow was triggered multiple times during development, but will only start working regularly when this change makes it all the way to master.

### CODE REVIEW

File: publish/deployed/mainnet/synths.json [hunk 1]
```diff
@@ -214,9 +214,9 @@
 		"asset": "BTC",
 		"subclass": "PurgeableSynth",
 		"inverted": {
-			"entryPoint": 9659,
```
Comment (justinjmoses) [inline publish/deployed/mainnet/synths.json:217]: shouldn't this be backported separately?

**Review by justinjmoses — DISMISSED** [review]
Just one comment about the separate backport

File: publish/deployed/mainnet/synths.json (hunk 1 above)
Comment (ajsantander) [reply publish/deployed/mainnet/synths.json]: Yes, absolutely.

Comment (ajsantander) [reply publish/deployed/mainnet/synths.json]: Ah, @justinjmoses I remember why this was here. Without this change, `job-validate-deployments-mainnet` fails: https://app.circleci.com/pipelines/github/Synthetixio/synthetix/4524/workflows/7fad6145-5fb9-4817-9952-5f87c118ea9c/jobs/33299

Comment (ajsantander) [reply publish/deployed/mainnet/synths.json]: The correct alternative, I guess, is to wait till the backport is on develop, and then update this branch. Here's the backport: https://github.com/Synthetixio/synthetix/pull/859

**Comment by ajsantander** [comment]
Closing this because a PR that branched off `develop-refactor-polish-1` was merged first: https://github.com/Synthetixio/synthetix/pull/861

**Pull request closed without merging**